api/v2/stable/index.html

```diff
@@ -1,44 +1,18 @@
-    });
 
-    if (false) {
-        var changed = null
-        var xhrChanged = new XMLHttpRequest();
-        var xhrData = new XMLHttpRequest();
-        setInterval(function() {
-            xhrChanged.open('GET', window.location.href + "/changed", true);
-            xhrChanged.send();
+<p>You probably arrived here looking for the NGSIv2 Specification document :)</p>
 
-            xhrChanged.onreadystatechange = function() {
-                if (xhrChanged.readyState == 4 && xhrChanged.status == 200) {
-                    if (!changed) {
-                        changed = xhrChanged.responseText
-                    }
-                    if (changed != xhrChanged.responseText) {
-                        changed = xhrChanged.responseText
-                        xhrData.open('GET', window.location.href + "/source", true);
-                        xhrData.send();
+<p>Instead of that, you should use the <a href="TBD">Orion API Specification</a>, which is based in the original NGSIv2 specification
```

Comment:
```suggestion
<p>Instead of that, you should use the <a href="https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md">Orion API Specification</a>, which is based in the original NGSIv2 specification
```